home/templates/blocks/media.html

```diff
@@ -0,0 +1,23 @@
+{% load wagtailcore_tags i18n %}
+
+{% if value.type == 'video' %}
```

Comment:
My preference would be to avoid conditionals in templates, if possible, because it's likely to be harder to test. For example, I see three separate templates here:
- media.html
- media-video.html
- media-audio.html

The main media template would use a more specific template depending on the type, but the switching would happen based on naming convention. Otherwise, perhaps the template to use can be decided within `MediaBlock`, where it may be easier to test.

I don't expect this to be changed, I am just making my preferences known.

Reply:
Fixed it!